crates/rattler_solve/src/libsolv_c/mod.rs

```diff
@@ -96,6 +96,12 @@ impl super::SolverImpl for Solver {
                 "timeout".to_string()
             ]));
         }
+        // Could ignore the strict_channel_priority flag here or do something like below maybe?
```

Comment:
This part I'm not so sure what we'd want the behavior to be, it seems to me that this libsolv code doesn't enforce strict channel priority? If so, we can either ignore the option all together to throw if strict priority is true since that's "not supported"?

Reply:
Yep. Probably we need to set the `priority` on the repository, though? I'll take a look.